plutus-tx/src/PlutusTx/Builtins/Internal.hs

```diff
@@ -319,6 +319,26 @@ emptyString = BuiltinString Text.empty
 equalsString :: BuiltinString -> BuiltinString -> BuiltinBool
 equalsString (BuiltinString s1) (BuiltinString s2) = BuiltinBool $ s1 == s2
 
+{-# NOINLINE showInteger #-}
+showInteger :: Integer -> BuiltinString
+showInteger = BuiltinString . Text.pack . show
+
+{-# NOINLINE showBool #-}
+showBool :: Bool -> BuiltinString
+showBool = BuiltinString . Text.pack . show
+
+{-# NOINLINE showString #-}
+showString :: BuiltinString -> BuiltinString
+showString (BuiltinString s) = BuiltinString . Text.pack $ show s
```

Comment:
this is a bit weird, shouldn't this just be id?

Reply:
Ah right, it adds quotes, right? In that case I question your use of it in the implementation of showing lists. Maybe we need some tests to check that these are doing the right thing!

Reply:
> I question your use of it in the implementation of showing lists

It uses `showString`, not `show @BuiltinString` :wink: 
There will be tests in the next PR

Reply:
Oh I guess it's a bit confusing since there are two `showString`s. Hmm. I'm not sure how to rename either one. Any suggestions?

Reply:
I don't have a good idea of renaming either one. `PlutusTx.Show.TH.showString` is consistent with base, and `PlutusTx.Builtins.Internal.showString` is consistent with the naming of other functions in that module.

The one in the implementation of show lists uses the former, so it doesn't add quotes.

Reply:
`PlutusTx.Show.TH.showsString`? It's also frankly suspicious: it does *not* show the string! it's more like `pure` or something

Reply:
> it does not show the string! it's more like pure or something

Yeah, it's basically the same as `Prelude.showString`. Converts a string into `ShowS`.